sql/core/src/test/resources/sql-tests/inputs/window.sql

```diff
@@ -239,6 +264,11 @@ SELECT
 	employee_name,
 	department,
 	salary,
+	FIRST_VALUE(employee_name) OVER  (
```

Comment:
we can use the named window frame syntax to avoid duplicating the window frame definition.

Reply:
OK